common/depot/helpers.go

```diff
@@ -0,0 +1,21 @@
+package depot
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+)
+
+const (
+	depotReposPath = "/repos"
+)
+
+var (
+	httpClient = &http.Client{Timeout: 10 * time.Second}
```

Comment:
Whoooops my bad,  I thought I linked to where it was used. It's used in `check_refs.go` line 23. Although I removed it as a global var and placed it inline.